regulations/static/regulations/css/less/module/comment.less

```diff
@@ -4,6 +4,40 @@ comment.less
 comment.less contains styles related to commenting on sections
 */
 
+#preamble-read {
+  margin: 0;
+  padding: 0;
+}
+
+.preamble-header {
+  position: absolute;
+  right: 0;
+  z-index: 2000;
```

Comment:
I forgot to remove that z-index line, but there are different z-indexes across some of the elements that I was testing.